File: test_db.py
import mysql.connector

try:
    mydb = mysql.connector.connect(
      host="localhost",
      user="root",
      password="tiger"
    )
    mycursor = mydb.cursor()
    print("--- SUPER ADMINS ---")
    mycursor.execute("SELECT super_admin_id, email FROM multimedia_governance.super_admin;")
    for x in mycursor.fetchall():
        print(x)
        
    print("--- USER DETAILS ---")
    mycursor.execute("SELECT user_id, email, super_admin_id FROM multimedia_governance.user_details;")
    for x in mycursor.fetchall():
        print(x)
except Exception as e:
    print(e)

File: TestDB.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.Statement;

public class TestDB {
    public static void main(String[] args) {
        try {
            Connection conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/multimedia_governance", "root", "tiger");
            Statement stmt = conn.createStatement();
            
            System.out.println("--- SUPER ADMINS ---");
            ResultSet rs = stmt.executeQuery("SELECT super_admin_id, email FROM super_admin");
            while (rs.next()) {
                System.out.println(rs.getString(1) + " | " + rs.getString(2));
            }
            
            System.out.println("--- USER DETAILS ---");
            rs = stmt.executeQuery("SELECT user_id, email, super_admin_id FROM user_details");
            while (rs.next()) {
                System.out.println(rs.getString(1) + " | " + rs.getString(2) + " | " + rs.getString(3));
            }
            
            conn.close();
        } catch(Exception e) {
            e.printStackTrace();
        }
    }
}
